docs/building.md

```diff
@@ -2,6 +2,6 @@
 
 The process for building Atmosphère is similar to building Fusée Gelée payloads and other Switch apps.
 
-In order to build Atmosphère you must have devkitPro installed on your computer. You can find instructions on how to install and setup devkitPro on various OSes here.
+In order to build Atmosphère you must have devkitPro installed on your computer. You can find instructions on how to install and setup devkitPro on various OSes [here](https://devkitpro.org/wiki/Getting_Started).
```

Comment:
This is very wrong. devkitPro is **not** software. devkitPro is an *organization* that produces and provides tools. You need to install devkitARM and devkitA64, which were produced by devkitPro. As the Getting Started page says:

> **Please note: devkitPro is the organisation that provides the tools. We are not a software package, we don't have version numbers and the only way to have us compile your code is to pay us (or maybe if you ask nicely when you need help figuring out an issue)**

Reply:
Fixed. I actually had already changed that in the Google Doc, but I forgot to update it on the repo, my bad.

Reply:
Alright, thank you for your speedy correction. I still do not understand why people get this wrong so often...